pkg/tasks/prometheusadapter.go

```diff
@@ -130,7 +130,27 @@ func (t *PrometheusAdapterTask) Run() error {
 		}
 	}
 	{
-		dep, err := t.factory.PrometheusAdapterDeployment()
+		apiAuthConfigmap, err := t.client.GetAPIServerAuthenticationConfigmap()
+		if err != nil {
+			return errors.Wrap(err, "failed to load kube-system/extension-apiserver-authentication configmap")
```

Comment:
It's a little messy specifying the string `kube-system/extension-apiserver-authentication` which is actually a const in another package. This task knows nothing about that namespace/name combination and should only say something like `failed to load the apiserver authentication configuration`. The specifics of what resource name are not a concern of this package, which is clear because the details are not even known (the package doesn't have the const). Makes sense?

Reply:
yep, sounds good :+1: